src/core/proj/qgscoordinatereferencesystem.cpp

```diff
@@ -1495,6 +1495,29 @@ QgsRectangle QgsCoordinateReferenceSystem::bounds() const
   return rect;
 }
 
+QString QgsCoordinateReferenceSystem::toOgcUri() const
+{
+  const auto parts { authid().split( ':' ) };
+  if ( parts.length() == 2 )
+  {
+    if ( parts[0] == QLatin1String( "EPSG" ) )
+      return  QStringLiteral( "http://www.opengis.net/def/crs/EPSG/9.6.2/%1" ).arg( parts[1] ) ;
```

Comment:
the usual practice is to use the dummy 0 generic version:
```suggestion
      return  QStringLiteral( "http://www.opengis.net/def/crs/EPSG/0/%1" ).arg( parts[1] ) ;
```

http://www.opengis.net/def/crs/EPSG/0/4326 is a dereferencable URL, whereas http://www.opengis.net/def/crs/EPSG/9.6.2/4326 is not

Reply:
Ok thanks, and I suppose for the OGC url the same?

Reply:
> and I suppose for the OGC url the same?

hum, I'm more used in seeing 1.3 being used... But both http://www.opengis.net/def/crs/OGC/0/CRS84 and http://www.opengis.net/def/crs/OGC/1.3/CRS84 work